ipn/ipnlocal/local.go

```diff
@@ -2442,6 +2444,7 @@ func (b *LocalBackend) popBrowserAuthNow() {
 	url := b.authURL
 	b.interact = false
 	b.authURL = "" // but NOT clearing authURLSticky
+	b.authURLTime = time.Time{}
```

Comment:
I think this should only reset with `authURLSticky`

Reply:
Done.